pageserver/src/tenant/timeline.rs

```diff
@@ -1422,7 +1422,8 @@ impl Timeline {
                     EvictionTaskTimelineState::default(),
                 ),
             };
-            result.repartition_threshold = result.get_checkpoint_distance() / 10;
+            result.repartition_threshold =
+                result.get_checkpoint_distance() / result.get_compaction_threshold() as u64;
```

Comment:
My understanding was not that this would be number of L0's we require for compaction, but that we do repartitioning about 10 times per checkpoint distance. They just happened to be the same number.

So with this change we would be conflating the two settings unnecessarily without any added clarification, `apples / 10` is now `apples / oranges` with added assumption `oranges == 10`.

If I've misunderstood then please let me know how is the repartitioning tied to number of L0's or why should it be?

Reply:
> we need a new configuration for that

Unsure if configuration should be added before we have a reason to adjust it in test. I think that most if not all of the settings are or were at one point covered by a test which tunes the value. Well, maybe not `max_lsn_wal_lag` (#4279) :)

Reply:
Hmm, then let's just make it a global constant so that people can easily know this is something changeable.